docs/user-guide/a1-06-spirv-target-specifics.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: user-guide
+permalink: /user-guide/target-specifics
```

Comment:
/user-guide/spirv-target-specific

Instead of putting this to appendix, let's create a chapter for "Target-specific Features", and make this nested under that.

Reply:
In the future we will also be adding a page for metal and webgpu, all under the same chapter.